assemblies/resources/bin/check_ports

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
This script is not executed but included. It shouldn't habe a shebang and should not be executable:

```
% ls -lh bin
total 40K
-rwxrwxr-x. 1 lars lars 3.1K Aug 26 21:19 check_ports
-rw-r--r--. 1 lars lars 9.6K Aug 26 21:24 inc
-rw-rw-r--. 1 lars lars 1.8K Aug 26 21:19 setenv
…
```

Reply:
fixed